app/helpers/compute.js

```diff
@@ -0,0 +1 @@
+export { default, compute } from 'ember-functional-helpers/helpers/compute';
```

Comment:
Can we also export in https://github.com/DockYard/ember-functional-helpers/blob/remove-compose/addon/index.js please